docs/driver_installation_zadig.md

```diff
@@ -0,0 +1,38 @@
+# Bootloader Driver Installation with Zadig
+
+QMK presents itself to the host as a regular HID keyboard device, and as such requires no special drivers. However, in order to flash your keyboard on Windows, the bootloader device that appears when you reset the board often *does*. There is one notable exception: the Caterina bootloader, usually seen on Pro Micros, appears as a serial port instead, and does not require a driver.
+
+We recommend the use of the [Zadig](https://zadig.akeo.ie/) utility. If you have set up the development environment with Msys2 or WSL, the `qmk_install.sh` script will have asked if you want it to install the drivers for you.
+
+## Installation
+
+Place your keyboard into the bootloader mode, either by hitting the `RESET` keycode (which may be on a different layer), or by pressing the reset switch usually located on the underside of the board. If your keyboard has neither, try holding Escape, or Space+`B`, as you plug it in (see the [Bootmagic](feature_bootmagic.md) docs for more details).
+
+Zadig will automatically detect the bootloader device:
+ - For keyboards with Atmel AVR MCUs, the bootloader will be named something similar to `ATm32U4DFU`.
+ - AVR keyboards flashed with the QMK-DFU bootloader will be named `<keyboard name> Bootloader`.
+ - For most ARM keyboards, it will be called `STM32 BOOTLOADER`.
+
```

Comment:
AVR Keyboards with USBaspbootloader name is `USBasp`

Reply:
Do they need one of the drivers provided by Zadig?

Reply:
Yes usually libusb-win32 but some people said that didnt work and they had to use libusbk

Reply:
Done